client/src/extension.ts

```diff
@@ -32,7 +32,7 @@ export function activate(context: ExtensionContext): void {
 
 // Get options for running Node language server
 function getServerOptions(context: ExtensionContext): ServerOptions {
-  const serverModule = context.asAbsolutePath(path.join('server', 'out-min', 'main.js'));
+  const serverModule = context.asAbsolutePath(path.join('server', 'out', 'server.js'));
```

Comment:
we'll just want to make sure to revert most of these changes to get it back into a working state before merging.